ci/edmtool.py

```diff
@@ -217,14 +212,25 @@ def install(runtime, toolkit, pillow, environment, source):
         source_pkgs = [
             github_url_fmt.format(pkg) for pkg in source_dependencies
         ]
+        # Without the --no-dependencies flag such that new dependencies on
+        # master are brought in.
         commands = [
-            "python -m pip install {pkg} --no-deps".format(pkg=pkg)
+            "python -m pip install --force-reinstall {pkg}".format(pkg=pkg)
             for pkg in source_pkgs
         ]
         commands = [
             "edm run -e {environment} -- " + command for command in commands
         ]
         execute(commands, parameters)
+
+    # No matter what happens before, always install local source again with no
+    # dependencies or we risk testing against an released enable.
+    install_local = (
+        "edm run -e {environment} -- "
+        "pip install --force-reinstall --no-dependencies " + ROOT
+    )
+    execute([install_local], parameters)
+
```

Comment:
The original location for this command is probably okay as none of apptools, pyface, traitsui and traits are going to install enable/kiva. Nevertheless I find it more reassuring that it is moved down here as safe measures.